bootstrap.sh

```diff
@@ -52,10 +62,17 @@ sanity(){
 
     # Ensure Nix has already been installed
     if [[ ! $(type nix-env 2>/dev/null) ]]; then
-        echo -e "Cannot find "$YELLOW"nix-env"$ESC" in the PATH"
-        echo "Please ensure you've installed and configured your Nix environment correctly"
-        echo -e "Install instructions can be found at: "$BLUE_UL"https://nixos.org/nix/manual/#chap-quick-start"$ESC""
-        exit 1
+        if [ ! -z $INSTALL_NIX ]; then
+            echo "Installing Nix ..."
+            # from https://nixos.org/nix/manual/#chap-quick-start
+            curl https://nixos.org/nix/install | sh || exit 1
+            . /nix/var/nix/profiles/default/etc/profile.d/nix-daemon.sh
```

Comment:
This apparently isn’t enough to load a newly-installed Nix into the current shell, so bootstrapping nix-darwin doesn’t work after this.

Reply:
It should be, maybe the daemon needs a second to start up?